recipes/numpy/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+cat > site.cfg <<'siteconfig'
+[DEFAULT]
+library_dirs = $PREFIX/lib
+include_dirs = $PREFIX/include
+
+[atlas]
+atlas_libs = openblas
+libraries = openblas
+
+[openblas]
+libraries = openblas
+library_dirs = $PREFIX/lib
+include_dirs = $PREFIX/include
+siteconfig
```

Comment:
Sure. Though I was thinking about including them in the `blas` package instead. That way we could avoid adding this sort of non-sense to multiple recipes and have it cleanly in one place.